config/async/100-async-redis-source.yaml

```diff
@@ -4,8 +4,8 @@ metadata:
   name: mystream
   namespace: knative-serving
 spec:
-  address: "rediss://user:userpassword@asdf1234.databases.appdomain.cloud:30285"
-  # address: "rediss://redis.redis.svc.cluster.local:6379"
+  #address: "rediss://user:userpassword@asdf1234.databases.appdomain.cloud:30285"
```

Comment:
Is this running locally? If yes, can the user and password be determined automatically so as not to add it to the config?